devicon.json

```diff
@@ -3415,5 +3415,26 @@
         },
         "color": "#68b604",
         "aliases": []
+    },
+    {
+        "name": "jest",
```

Comment:
If the folder name is `jestjs`, this should be `jestjs`.
This will fix the [workflow](https://github.com/devicons/devicon/pull/527/checks?check_run_id=2050970616#step:7:17) checks to stop failing.